contracts/JBveBanny.sol

```diff
@@ -344,6 +354,11 @@ contract JBveBanny is ERC721Votes, ERC721Enumerable, Ownable, ReentrancyGuard, J
     bytes memory _metadata,
     IJBPayoutRedemptionPaymentTerminal _terminal
   ) external nonReentrant {
+    {
+      // Check the permissions scoped to prevent stack too deep
+      _requirePermission(ownerOf(_tokenId), projectId, JBStakingOperations.REDEEM);
+    }
```

Comment:
I believe any user could redeem any other user's tokens since the caller wasn't getting checked

Reply:
the permission check happens in Terminal. The token holder has to give the veBanny operator permission to redeem its tokens on its behalf.

Reply:
worth a comment explaining this inline